src/Microsoft.ML.Data/Evaluators/MultiClassClassifierEvaluator.cs

```diff
@@ -998,16 +998,17 @@ private protected override IEnumerable<string> GetPerInstanceColumnsToSave(RoleM
         // Multi-class evaluator adds four per-instance columns: "Assigned", "Top scores", "Top classes" and "Log-loss".
         private protected override IDataView GetPerInstanceMetricsCore(IDataView perInst, RoleMappedSchema schema)
         {
-            // If the label column is a key without key values, convert it to I8, just for saving the per-instance
+            // If the label column is a key without text key values, convert it to I8, just for saving the per-instance
```

Comment:
I noticed that the code actually converts to R8, not I8.

Reply:
Ahhh. Delightful... seems a pity to update such a simple PR for an internal class. How about this, I'll locally fix it, and if for some other reason I need to update the PR it'll go in, otherwise it'll just go into my next PR.